pkg/pillar/cmd/zedmanager/handlevolumemgr.go

```diff
@@ -224,13 +213,12 @@ func getVolumeRefConfigFromAIConfig(config *types.AppInstanceConfig,
 	log.Tracef("getVolumeRefConfigFromAIConfig(%v)", vrs.Key())
 	for i := range config.VolumeRefConfigList {
 		vrc := &config.VolumeRefConfigList[i]
-		if vrc.VolumeID == vrs.VolumeID &&
-			vrc.GenerationCounter == vrs.GenerationCounter &&
-			vrc.LocalGenerationCounter == vrs.LocalGenerationCounter {
```

Comment:
Is the LocalGenerationCounter used when the operator purges a volume from using the Local Profile Server API? If so, can we remove it?

And is there any harm in keeping the check on GenerationCounter until we deprecate the generation counter from the EVE API? (That might require some more care.)

Reply:
ah - got it.